Problem 1

```diff
@@ -0,0 +1,44 @@
+# Implement Trie - Prefix tree
+
+#Tc:O(m)
```

Comment:
What is  m here . Also check TC once , length of the. word will also be included